RecoBTag/TensorFlow/test/test_deep_doublec_cfg.py

```diff
@@ -0,0 +1,91 @@
+
+import FWCore.ParameterSet.Config as cms
+from PhysicsTools.PatAlgos.tools.helpers import getPatAlgosToolsTask
```

Comment:
it looks like this file includes a test of pfDeepDoubleBJetTags as well.
I suggest to rename this to ```test_deep_double_cfg.py``` and remove ```test_deep_doubleb_cfg.py```